packages/icons/src/SVGBase/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SVG = ({
+  children, colour, title, ...rest
+}) => (
+  <svg
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    x="0px"
+    y="0px"
+    width="100%"
+    height="100%"
+    style={{ display: 'block' }}
+    fill={colour}
```

Comment:
where we are passing properties down as-is, unless there is a reason not to, I would persist the prop name, so in this case I would call the prop `fill`